cd2h_repo_project/config.py

```diff
@@ -48,6 +49,29 @@ def _(x):
     return x
 
 
+# Flask configuration
+# ===================
+# See details on
+# http://flask.pocoo.org/docs/0.12/config/#builtin-configuration-values
```

Comment:
Update to version `1.1.x`?

Reply:
Good idea. I actually linked to the master version. This way the comment won't go stale.

Reply:
Nice, even better.